src/integration/kotlin/pl/edu/agh/gem/integration/controller/ExternalUserDetailsControllerIT.kt

```diff
@@ -201,5 +211,69 @@ class ExternalUserDetailsControllerIT(
                 errors.first().code shouldBe MissingUserDetailsException::class.simpleName
             }
         }
+
+        should("update user details successfully") {
+            // given
+            val user = createGemUser(USER_ID)
+            val updateRequest = createUserDetailsUpdateRequest(USER_ID)
+            userDetailsRepository.save(createUserDetails(USER_ID))
+
+            // when
+            val response = service.updateGroupUserDetails(user, updateRequest)
+
+            // then
+            response shouldHaveHttpStatus OK
+        }
+
+        should("return NOT_FOUND when user details doesn't exist") {
```

Comment:
that case can happen ? ( user dont have your own details )

Reply:
zrobić zeby rzucało 5XX